services/brig/src/Brig/IO/Intra.hs

```diff
@@ -623,26 +638,28 @@ getTeamContacts u = do
 -- body.  When brig wants to know, it can call both parts.  These thoughts may all become obsolete
 -- if we introduce a deletion service in the future.
 getTeamOwners :: TeamId -> AppIO [Team.TeamMember]
```

Comment:
This function now doesn't do filtering by email, but `getTeamOwnersWithEmail` does.